server/stream.go

```diff
@@ -1124,11 +1133,6 @@ func (s *Server) checkStreamCfg(config *StreamConfig, acc *Account) (StreamConfi
 				if cfg.MaxMsgSize > 0 && maxMsgSize > 0 && cfg.MaxMsgSize < maxMsgSize {
 					return StreamConfig{}, NewJSSourceMaxMessageSizeTooBigError()
 				}
-				if !isRecovering && !hasFilterSubjectOverlap(src.FilterSubject, streamSubs) {
```

Comment:
Any way to check if this is misconfigured here vs support call on why aren't my messages synching?

Reply:
Not easily, and I don't think it would be foolproof and even then it comes down to the same argument as above in that things can change over time and what is valid today may not be tomorrow and vice-versa. 

In terms of mis-configuration in `nats s info` gives you the filter for each source defined so you can then create a consumer yourself on that stream with the same filter and see if you get new messages or not and if not look at that sourced stream without a filter to see what the subjects look like.